src/app/tracing/util/constants.ts

```diff
@@ -1,9 +1,35 @@
 import { Color, GraphType, MapType, DeliveryData, StationData } from '../data.model';
 import { List, Map } from 'immutable';
+import { ExampleData } from '@app/main-page/model/types';
 
 export class Constants {
     static readonly EXAMPLE_MODEL_FILE_PATH = 'assets/data/ExampleData.json';
```

Comment:
Is this constant still required? If not please remove.

Reply:
It is still used in the test data-importer-v1.spec.ts. The test needs to be updated in order to remove this constant.
Ticket #644 is created.